src/RegistrationController.js

```diff
@@ -147,11 +151,16 @@ function (
         var name = schemaProperty.get('name');
         if (name === 'password' || name === 'login') {
           inputOptions.events = {
-            'input': function() {
+            'input': function () {
               checkPasswordMeetComplexities(this.model);
             }
           };
         }
+        if (name === 'password') {
+          inputOptions.events.focusout = function () {
```

Comment:
Thanks! Firefox only recently introduced support for this event in version 52 (https://developer.mozilla.org/en-US/docs/Web/Events/focusout#Browser_compatibility). To be on the safe side, would it be possible to use ``blur``?